<ui version="4.0" >
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>448</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>448</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Stretch IDE</string>
  </property>
  <widget class="QLabel" name="label_1" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>12</y>
     <width>231</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Digital Video Recorder</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>47</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Control Panel Version:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>67</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SDK Version:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sdk_version" >
   <property name="geometry" >
    <rect>
     <x>338</x>
     <y>67</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelLogFilePath" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>265</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Log Files Path:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLEditLogFilePath" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>263</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>318</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;OK</string>
   </property>
   <property name="shortcut" >
    <string>Alt+O</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>294</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Copyright (c) 2008 Stretch, Inc. All Rights Reserved.</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dvrcp_version" >
   <property name="geometry" >
    <rect>
     <x>338</x>
     <y>47</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>161</x>
     <y>83</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Board 1 Information:</string>
   </property>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>142</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Chip Revision:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dvr_chip_rev" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>142</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>121</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>DVR Board Type-Rev:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dvr_board_type_rev" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>121</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>101</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>BSP Version:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bsp_version" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>101</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>81</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Boot Loader Version:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="boot_loader_version" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>81</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Firmware Build Date:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fw_build_date" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>60</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>41</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Firmware Version:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>21</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>PCIe Driver Version:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fw_version" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>41</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="driver_version" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>21</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>AboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>186</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>79</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
